native/cocos/renderer/gfx-vulkan/VKTexture.cpp

```diff
@@ -43,24 +43,10 @@ CCVKTexture::~CCVKTexture() {
 
 void CCVKTexture::doInit(const TextureInfo & /*info*/) {
     _gpuTexture = ccnew CCVKGPUTexture;
-    _gpuTexture->type = _info.type;
-    _gpuTexture->format = _info.format;
-    _gpuTexture->usage = _info.usage;
     _gpuTexture->width = _info.width;
     _gpuTexture->height = _info.height;
-    _gpuTexture->depth = _info.depth;
-    _gpuTexture->arrayLayers = _info.layerCount;
-    _gpuTexture->mipLevels = _info.levelCount;
-    _gpuTexture->samples = _info.samples;
-    _gpuTexture->flags = _info.flags;
     _gpuTexture->size = _size;
-
-    cmdFuncCCVKCreateTexture(CCVKDevice::getInstance(), _gpuTexture);
-
-    if (!_gpuTexture->memoryless) {
-        CCVKDevice::getInstance()->getMemoryStatus().textureSize += _size;
-        CC_PROFILE_MEMORY_INC(Texture, _size);
-    }
+    createTexture();
```

Comment:
I think it is not good to initialize _gpuTexture in two functions: `doInit` and `createTexture`.